src/components/menu/menu-item/menu-item.component.js

```diff
@@ -61,7 +187,8 @@ const MenuItem = ({
     icon,
     hasSubmenu: Boolean(submenu),
     selected,
-    menuType
+    menuType,
+    tabbable: !!isFirstElement
```

Comment:
`isFirstElement` is a bool, no need to coerce the value.
```suggestion
    tabbable: isFirstElement
```